DatadogCore/Tests/Datadog/RUM/RUMMonitorTests.swift

```diff
@@ -30,6 +30,53 @@ class RUMMonitorTests: XCTestCase {
         super.tearDown()
     }
 
+    // MARK: - Current Session Id
+    func testWhenSessionIsSampledIn_itReturnsCurrentSessionId() throws {
+        // Given
+        var capturedSession: String?
+        config.dateProvider = RelativeDateProvider(startingFrom: Date(), advancingBySeconds: 1)
+        config.sessionSampleRate = 100.0
+        config.onSessionStart = { session, sampled in
+            capturedSession = session
+        }
+        RUM.enable(with: config, in: core)
+        let monitor = RUMMonitor.shared(in: core)
+
+        monitor.startView(viewController: mockView)
```

Comment:
suggestion/ RUM session should start immediately, so there is no need to start a view. It might be worth changing it to cover the minimal behaviour.